src/sentry/search/snuba/executors.py

```diff
@@ -293,7 +294,6 @@ def query(
             if (
                 cursor is None
                 and sort_by == "date"
-                and not environments
```

Comment:
Just noting that I discussed this with @JTCunning and we did some queries, and filtering by env seems to perform just fine when doing postgres only queries, so we're removing this restriction.

Reply:
I reserve the right to take back my words. <3